CMakeLists.txt

```diff
@@ -490,6 +490,7 @@ if(USE_TCMALLOC)
         CXXFLAGS=${THIRD_PARTY_CXX_FLAGS}
     BUILD_COMMAND make
     BUILD_IN_SOURCE 0
+    BUILD_BYPRODUCTS <INSTALL_DIR>/lib/libtcmalloc_minimal.a
```

Comment:
Quick question: Does INSTALL_DIR refer to where libtcmalloc_minimal.a is installed? If so can we name this variable TCM_INSTALL_DIR? 

Thanks @d for getting involved!